src/candidatelist/service/candidate-list.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+
+import { Campaigns } from '../../models/common/campaign.model';
+
+const LEVERAGE_API: string = 'http://api.leveragecampaignfinance.org';
+
```

Comment:
Can we change this api call from the service to a proxy.ts file. This is a more appropriate place to be calling the back end.